docs/DEVELOPER-GUIDE.md

```diff
@@ -39,7 +43,8 @@
     * git remote add upstream git@github.com:appsody/appsody.git
```

Comment:
Given https://help.github.com/en/github/using-git/which-remote-url-should-i-use#cloning-with-https-urls-recommended is it not clearer to use:
`git remote add upstream https://github.com/appsody/appsody.git`
?
(This is certainly referenced in other tutorials on github.com such as https://help.github.com/en/github/using-git/removing-a-remote)